src/Compilers/Core/Portable/Operations/ILiteralExpression.cs

```diff
@@ -13,10 +13,6 @@ namespace Microsoft.CodeAnalysis.Semantics
     /// </remarks>
     public interface ILiteralExpression : IOperation
     {
-        /// <summary>
-        /// Textual representation of the literal.
-        /// </summary>
-        string Text { get; }
```

Comment:
What's the point in having an interface with no members? In general I'm suspicious of marker interfaces.